### ISSUE TITLE
Do the fully connected and convolution layers of the created KerAS model need to be initialized with fixed weights?

### ISSUE THREAD

**Issue by skiifall** [tensorflow/federated#2737]
The network I am training now consists of the convolution layer and the full connection layer, but without initializing the weight of the network, I find that the effect is not good. So my question is, is it necessary to initialize weights for the neural network layer? Meanwhile, I can't understand the working principle of Tff.learning.build_federated_averaging_process () well through the official documents, and I don't understand why the server side needs to set the extra learning rate instead of the same as the client side
